.github/ISSUE_TEMPLATE/1.bug.yml

```
@@ -8,20 +8,22 @@ body:
         Please provide a searchable summary of the issue in the title above ⬆️.
 
         Thanks for contributing by creating an issue! ❤️
-  - type: checkboxes
+
+  - type: input
     attributes:
-      label: Duplicates
-      description: Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem.
-      options:
-        - label: I have searched the existing issues
-          required: true
+      label: Search keywords
+      description: Which keywords did you use to search for duplicates? Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem.
```

NIT

not sure if this complicates things, but I would probably ask the question at the end of the description:
```suggestion
      description: Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem. Which keywords did you use to search for duplicates?
```

---

same for the other templates

---

@Janpot proposed this change, do you have a suggestion how to present this field? We should probably update the title to something like "Duplicate issues search keywords" to make it clear what we want from them. I'd keep the "Which keywords ...?" first though, because it indicates what we expect from them in this field.

Or simplify it to:

```
Which keywords did you use to search for duplicates in the [list of existing issues](https://github.com/mui/mui-x/issues)?
```

---

I think "Search keywords", or even just "keywords" works well as a title. To get them on board with the extra friction we could consider 
1. reminding them that there may already be a ticket and so they can spare themselves the effort of having to fill out the template altogether
1. emphasising that providing the keywords benefits other users (and therefore their future self as well).

How do you feel about something along the lines of

> Your issue may already have been reported. Please list the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will allow us to improve the experience for other users.

---

Applied.